src/fb/mod.rs

```diff
@@ -0,0 +1,367 @@
+    };
+    pub const PAYLOAD: Self = Self {
+        transport: false,
+        payload: true,
+    };
+}
+
+impl std::ops::BitOr<FciFeedbackPacketType> for FciFeedbackPacketType {
+    type Output = Self;
+    fn bitor(self, rhs: Self) -> Self::Output {
+        Self {
+            transport: self.transport | rhs.transport,
+            payload: self.payload | rhs.payload,
+        }
+    }
+}
+
+impl std::ops::BitAnd<FciFeedbackPacketType> for FciFeedbackPacketType {
+    type Output = Self;
+    fn bitand(self, rhs: Self) -> Self::Output {
+        Self {
+            transport: self.transport & rhs.transport,
+            payload: self.payload & rhs.payload,
+        }
+    }
+}
+
+/// A parsed (Transport) Feedback packet as specified in RFC 4585.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct TransportFeedback<'a> {
+    data: &'a [u8],
+}
+
+impl<'a> RtcpPacket for TransportFeedback<'a> {
+    const MIN_PACKET_LEN: usize = 12;
+    const PACKET_TYPE: u8 = 205;
+}
+
+impl<'a> RtcpPacketParser<'a> for TransportFeedback<'a> {
+    fn parse(data: &'a [u8]) -> Result<Self, RtcpParseError> {
+        parser::check_packet::<Self>(data)?;
+        Ok(Self { data })
+    }
+
+    #[inline(always)]
+    fn header_data(&self) -> [u8; 4] {
+        self.data[..4].try_into().unwrap()
+    }
+}
+
+impl<'a> TransportFeedback<'a> {
+    pub fn builder() -> TransportFeedbackBuilder {
+        TransportFeedbackBuilder::default()
+    }
+
+    pub fn padding(&self) -> Option<u8> {
+        parser::parse_padding(self.data)
+    }
+
+    pub fn sender_ssrc(&self) -> u32 {
+        parser::parse_ssrc(self.data)
+    }
+
+    pub fn media_ssrc(&self) -> u32 {
+        parser::parse_ssrc(&self.data[4..])
+    }
+
+    pub fn parse_fci<F: FciParser<'a>>(&self) -> Result<F, RtcpParseError> {
+        if F::PACKET_TYPE & FciFeedbackPacketType::TRANSPORT == FciFeedbackPacketType::NONE {
+            return Err(RtcpParseError::WrongImplementation);
+        }
+        if parser::parse_count(self.data) != F::FCI_FORMAT {
+            return Err(RtcpParseError::WrongImplementation);
+        }
+        F::parse(&self.data[12..])
+    }
+}
+
+/// TransportFeedback packet builder
+#[derive(Debug, Default)]
+#[must_use = "The builder must be built to be used"]
+pub struct TransportFeedbackBuilder {
+    padding: u8,
+    sender_ssrc: u32,
+    media_ssrc: u32,
+    fci: Option<Box<dyn FciBuilder<'static>>>,
+}
+
+impl TransportFeedbackBuilder {
+    pub fn sender_ssrc(mut self, sender_ssrc: u32) -> Self {
+        self.sender_ssrc = sender_ssrc;
+        self
+    }
+
+    pub fn media_ssrc(mut self, media_ssrc: u32) -> Self {
+        self.media_ssrc = media_ssrc;
+        self
+    }
+
+    /// Sets the number of padding bytes to use for this TransportFeedback packet.
+    pub fn padding(mut self, padding: u8) -> Self {
+        self.padding = padding;
+        self
+    }
+
+    pub fn fci(mut self, fci: impl FciBuilder<'static> + 'static) -> Self {
```

Comment:
Since `fci` is mandatory, I would use it as an argument to the `builder()` constructor, so we don't need to use an `Option` and check its presence later (e.g. in `calculate_size()`).